source/_patterns/20-molecules/embed/embed.scss

```diff
@@ -1,15 +1,48 @@
 .embed-inline {
   position: relative;
   margin: 32px 0;
-  display: flex;
+  display: block;
   align-items: center;
   justify-content: center;
 
   @include smartphone {
-    margin: 24px 0x;
+    margin: 24px 0px;
   }
 }
 
-.embed-inline iframe {
-  display: none;
+.embed-inline__title {
+  padding: 0 16px;
+  font-family: $font-medium;
+  margin-bottom: 12px;
+
+  @include smartphone {
+    padding: 0 24px;
+    margin-bottom: 8px;
+  }
+}
+
+.embed-inline__content > * {
+  margin: 0 auto;
+}
+
+body[data-representation=web] {
```

Comment:
Da bin nicht hundert Prozent sicher dass ich es nur in Smartphone Ansicht machen soll (muss es noch mit Daniel absprechen ob auch Tablet view inbegriffen ist)